cmd/_integration-tests/Makefile

```diff
@@ -7,20 +7,28 @@ TRANSPORT_TEST_MSG=\n$(OK_COLOR)Starting transport end to end test:$(END_COLOR_L
 
 CLI_TEST_MSG=\n$(OK_COLOR)Start server and cliclient generate, build, and run test:$(END_COLOR_LINE)
 
+export PATH := $(CURDIR):$(PATH)
 
 all: test
 
 test: clean test-transport test-cli
```

Comment:
This should probably have a clean at the end of it as well, otherwise these binaries are going to stick around even after the tests pass. Then there are hanging out in the repo untracked untell you call `make testclean`. If the tests pass, the binaries should be removed.

The tests in `cli` and `transport` both clean up at the end if the pass. Though we need something here to remove the binaries.